http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/service/LeonardoService.scala

```diff
@@ -1154,7 +1156,8 @@ class LeonardoService(protected val dataprocConfig: DataprocConfig,
     val dummyStagingBucket = GcsBucketName("dummy-staging-bucket")
     val dummyClusterRequest = ClusterRequest()
 
-    val clusterInit = ClusterInitValues(cluster.googleProject, cluster.clusterName, dummyInitBucket, dummyClusterRequest,
+    //TODO: why is staging bucket optional here?
```

Comment:
It's because `Cluster`'s `DataprocInfo` takes an optional `stagingBucket` and I think that's because we've overloaded the usage of `Cluster` objects. Early (during the synchronous part) in the cluster creation cycle, there's yet not `stagingBucket` defined but we still use `Cluster` to encapsulate the cluster fields.